schema.ts

```diff
@@ -799,9 +791,6 @@ export interface StringWithOptionsSchema
     PropertyWithAutocompleteWithOptionalDisplay<string> {
   /** Instructs Coda to render this value as a select list. */
   codaType: ValueHintType.SelectList;
-
-  /** @hidden */
-  autocomplete: PropertySchemaAutocompleteWithOptionalDisplay<string>;
```

Comment:
Note: removed the property because it's already defined by the `PropertyWithAutocompleteWithOptionalDisplay<string>` parent interface.